ecf/defs/prod00.def

```diff
@@ -3,30 +3,38 @@ extern /prod18/gdas/atmos/post
 extern /prod18/gdas/enkf/post
 #### ecen
 extern /prod18/gdas/atmos/post_processing/jgdas_atmos_chgres_forenkf
+extern /prod06/gdas/atmos/analysis/jgdas_atmos_analysis
+extern /prod06/gfs/atmos/analysis/jgfs_atmos_analysis
+extern /prod06/gdas/enkf/analysis/recenter/ecen
```

Comment:
I am curious what in the 00z suite is depending on a trigger from 06z.
I don't see any references to triggers from these.

As a matter of fact, the only triggers are from 
`/prod18/gdas/enkf/post` and `/prod18/gdas/atmos/post_processing/jgdas_atmos_chgres_forenkf`

Reply:
I had the same question. Why are new triggers being added when this is only supposed to be a port? And why triggers from three cycles previous?